.github/workflows/inshpect.yml

```diff
@@ -0,0 +1,46 @@
+name: Inshpect
+
+on:
+  pull_request:
+
+jobs:
+  inshpect:
+    runs-on: ubuntu-22.04
```

Comment:
Would be nice to run it already on ubuntu 24.04

Reply:
Perfect, I add this review!
Thanks @rasolca !

Reply:
You need `runs-on: ubuntu-24.04` (note 24, not 22). You of course need to install them with apt instead and since I haven't tried it myself the ubuntu versions may not be compatible, but give it a try and then we'll see if it works.

Reply:
Yes, sorry I made an mistake when I wrote `22.04`

I pushed my modify and my updates.